README.md

```diff
@@ -68,13 +83,16 @@ We follow [semantic versioning](https://semver.org/) and are guided by the princ
 We may occasionally make breaking changes in order to improve the user experience.
 When possible, we will keep old interfaces and mark them as deprecated, as long as they can co-exist with the
 new ones.
-Each substantial improvement, breaking change, or deprecation will be documented in the release notes.
+Each substantial improvement, breaking change, or deprecation will be documented in the
```

Comment:
I changed this because I'm not starting with any release notes. Leaving the entire repo history internal